.github/pull_request_template.md

```diff
@@ -20,15 +20,6 @@ Which parts of this PR were you unsure about? Which parts were particularly tric
 
 If you're stuck on part of the changes or want feedback early, open a draft PR and list the items that need to be completed here using a checklist.
 
-## Changelog
-
-> This section is optional. If this was a trivial fix, or has no externally-visible impact, you can delete this section.
-
-- What changed as a result of this PR?
-- If applicable, organize changes under "Added", "Changed", or "Fixed" sub-headings
-- Stick to one or two sentences. If more detail is needed for a particular change, consider adding it to the "Solution" section
-  - If you can't summarize the work, your change may be unreasonably large / unrelated. Consider splitting your PR to make it easier to review and merge!
-
```

Comment:
This is just my opinion, but I don't think `Changelog` is necessarily redundant. It's useful for identifying the key changes to a PR that might be hidden behind a wall of text in the other sections. It's the tl;dr.

Reply:
My personal feeling is that PR authors can and should do a better job highlighting this in Objective. If we're looking to reduce friction, this is a good place to cut.

Reply:
Fair enough. But I didn't see a lot of PRs doing that before. Maybe we call it out rather than making an entire section? Like, "Hey if this is a long PR, maybe consider adding a changelog section/list!"

If it's better completely removed, I understand but I'm in favor of at least the reminder haha